OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -104,7 +104,9 @@ EditorActorPreview Add(string id, ActorReference reference, bool initialSetup =
 
 			var preview = new EditorActorPreview(worldRenderer, id, reference, owner);
 			previews.Add(id, preview);
-			screenMap.Add(preview, preview.Bounds);
+
+			if (preview.Bounds.Width > 0 && preview.Bounds.Height > 0)
```

Comment:
What is the reasoning behind this?  It looks like a workaround for bogus ActorPreviews, which should be fixed in another way.